examples/next/lib/theme.ts

```diff
@@ -1,6 +1,6 @@
-import { makeTheme } from '@theme-ui/css/utils'
+// import { makeTheme } from '@theme-ui/css/utils'
```

Comment:
After running `pnpm build` Next.js dev server doesn't catch that the libraries are built instantly, so I removed `.next` dir in the example, restarted `pnpm dev` and the error on this import went away.

The error was basically saying we can't import TypeScript files directly, and that's totally fine, because we want to develop examples against prebuilt libraries, but even after rebuilding you gotta clean the cache if you started dev server previously.